zuul.d/jobs.yaml

```diff
@@ -2214,3 +2214,20 @@
       - OS:ubuntu-xenial
       - Arch:arm64
       - BuildType:Compile
+
+# Spark arm64 build and run unit tests periodic job
+- job:
+    name: spark-unit-test-arm64
```

Comment:
So we may change the job name to show this job is spark master build and test with hadoop 2.7 
The reason for introducing hadoop version, as the dependency libriries will have a huge change. If we can split the job, it's better to maintain for us.

Reply:
Done